roles/common/tasks/cr_tests.yml

```diff
@@ -7,14 +10,21 @@
   failed_when:
     - result.rc != 0
 
-- name: Verify that a CR is ready {{ common_cr_ready_test_id }}
+  # This is added so that the task name will be rendered correctly
+  # If item.condition_type is used and it doesn't exist, then the
+  # literal template text will appear as the task name
+  # e.g. "Verify that the {{ item.kind }} {{ item.name }} CR is {{ item.condition_type }}"
+- name: Set the condition type
+  ansible.builtin.set_fact:
+    condition_type: "{{ item.condition_type if item.condition_type is defined else '' }}"
+
+- name: |
+    Verify that the {{ item.kind }} {{ item.name }} CR is {{ condition_type }}
```

Comment:
The name makes less sense to me like this. It's expected for the condition_type to not be defined (to be set to '' by the task above) right? In that case the name will render into just "Verify that the <kind> <name> CR is".

The condition type is used to determine where to look in a CR to determine it's in some expected state. A not so practical example could be, that we want to check that the CeilometerReady is True in the Telemetry CR. Now the name would be: "Verify that the telemetry telemetry CR is CeilometerReady".

You could reword it to something like: "Verify that the {{ condition_type }} condition of {{ item.kind }} {{ item.name }} is True"